fe/fe-core/src/main/java/com/starrocks/sql/ast/AsyncRefreshSchemeDesc.java

```diff
@@ -10,20 +12,53 @@ public class AsyncRefreshSchemeDesc extends RefreshSchemeDesc {
 
     private LocalDateTime startTime;
 
-    private IntervalLiteral intervalLiteral;
+    private long step;
 
-    public AsyncRefreshSchemeDesc(LocalDateTime startTime, IntervalLiteral intervalLiteral) {
+    private TimestampArithmeticExpr.TimeUnit timeUnit;
```

Comment:
TimestampArithmeticExpr has nothing to do with here, why only use TimestampArithmeticExpr? Why not use interval?